### ISSUE TITLE
As a user, I would like to continue moving across results in preview mode also when the result page changes

### ISSUE THREAD

**Issue by lilimelgar** [beeldengeluid/labo-components#288]
Where: Search tool, result list, preview
Current situation: the user is moving across results from the list using the implemented functionality in issue [#250](https://github.com/beeldengeluid/labo-components/issues/250), when the user reaches the last result in a page, s(he) has to go back to the result list, change the page, and go back to preview. 

Acceptance criteria:
- [ ] It is possible to go back and forth between results in preview mode even if the result pages change. 

Note: This is possible in the Resource viewer (full view) mode. Is it possible to implement the same functionality in the preview?

**Comment by jblom** [comment]
@lilimelgar yes this was not implemented so far. I also waited with this, because doing it properly would entail reusing the paging code from the resource viewer, which takes a bit of time to do it properly. Moving it to the icebox for now though. First the more pressing things
